spid_cie_oidc/authority/schemas/trust_mark_status_endpoint.py

```diff
@@ -0,0 +1,14 @@
+
+from typing import Optional
+
+from pydantic import BaseModel, HttpUrl, constr
+
+
+class TrustMarkRequest(BaseModel):
+    sub : Optional[HttpUrl]
+    id : Optional[HttpUrl]
+    trust_mark : constr(regex=r"^[a-zA-Z\_\-0-9]+\.[a-zA-Z\_\-0-9]+\.[a-zA-Z\_\-0-9]+") # noqa: F722
```

Comment:
sub and id are requried if trust_mark is absent

trust mark is not mandatory is sub and id are present